packages/components/src/ListView/Items/Items.component.js

```diff
@@ -153,13 +153,15 @@ Items.propTypes = {
 	getItemHeight: PropTypes.func,
 	searchCriteria: PropTypes.string,
 	toggleAllChecked: PropTypes.bool,
+	toggleAllShown: PropTypes.bool,
```

Comment:
Can you rename it to match other props format ?
getItemHeight, searchCriteria, .. ---> `showToggleAll`